sdk/storage/Azure.Storage.Common/src/Shared/Errors.Clients.cs

```diff
@@ -85,4 +85,4 @@ public static void VerifyHttpsTokenAuth(Uri uri)
             }
         }
     }
```

Comment:
Looks like an accidental change that pulled in a bunch of other Code Owners on the PR.  Might be good to revert this file.